org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/thingTranslator.js

```diff
@@ -185,7 +189,9 @@ class Translator {
      * Updates the specified resource with the provided payload.
      */
     postDeviceResource(di, resourceId, payload) {
-        if (di === generateGUID( this.controlId + 'oic.d.smartplug' )) {
+        console.log(this.uuid + ":" + generateGUID( this.uuid + 'oic.d.smartplug' ));

```

Comment:
Fixed.

---
In reply to: [114233427](https://github.com/openT2T/translators/pull/237#discussion_r114233427) [](ancestors = 114233427)